infra/bin/app.ts

```diff
@@ -1,75 +1,134 @@
 #!/usr/bin/env node
 
 import * as cdk from "@aws-cdk/core";
-import { RdsStack, AppInfo } from "@rylabs/ry-cdk-tools/lib/rds";
+import { RdsStack } from "@rylabs/ry-cdk-tools/lib/rds";
 import { InstanceType, InstanceSize, InstanceClass } from "@aws-cdk/aws-ec2";
-import { RailsPipelineStack, RailsStack } from "@rylabs/ry-cdk-tools/lib/rails";
+import {
+  RailsPipelineStack,
+  RailsStack,
+  AppInfo,
+  Conventions,
+} from "@rylabs/ry-cdk-tools/lib/rails";
 
 const env = {
-  account: "Enter Your AWS Account ID",
-  region: "us-east-1",
+  account: "AWS_ACCOUNT_ID",
+  region: "us-east-2",
 };
 
 const appInfo: AppInfo = {
   // Name of the app, in lower title case format. Use for naming AWS resources.
-  name: "enterAppName",
+  name: "app name here",
 
   // Short name of the environment. Use for naming AWS resources.
-  environment: "envShortName",
+  environment: "staging",
 
   // Environment name.
-  orgName: "Enter Your Organization Name",
+  orgName: "Company name",
 
   // Author name.
-  author: "Enter Your Name",
+  author: "Your name here",
 };
 
 // Github repo owner name
-const ownerName = "enter-repo-owner-name";
+const ownerName = "Github repo owner name";
 
 // Github repo name
-const repoName = "enter-repo-name";
+const repoName = "Github repo name";
+
+// Branch this environment deploys from
+const branchName = "develop";
 
 // AWS VPC ID
-const vpcId = "enter-vpc-id";
+const vpcId = "vpc-ID#####";
+
+// Ruby stack
+const solutionStackName = "64bit Amazon Linux 2 v3.2.2 running Ruby 2.7";
 
-// Server EC2 Instance type
-const instanceType = InstanceType.of(InstanceClass.T2, InstanceSize.MEDIUM);
+const dbInstanceIdentifier = "app-staging";
+const dbInstanceEndpointAddress =
+  "AWS_RDS_ENDPOINT.rds.amazonaws.com";
+const dbPort = 5432;
+const dbName = "app_staging";
+const dbSecurityGroup = "sg-########";
+const dbUsername = "appstagingdbuser";
+const dbPassword = {
+  secretId: "appnamestagingdbMasterPasswor-######",
+  jsonField: "password",
+};
 
 const app = new cdk.App();
 const vpc = { vpcId };
+const conventions = new Conventions(appInfo);
 
-// Database with medium T2 instance
```

Comment:
I had some RDS instructions commented out that should be copied here.

I've started creating the companion documentation for this in the wiki, you can see what I'm thinking of about why I want those sections here: https://github.com/RYLabs/rails-starter/wiki/AWS